src/mokka/src/test/java/pl/hycom/mokka/stubbing/WireMockMappingSourceTest.java

```diff
@@ -0,0 +1,37 @@
+package pl.hycom.mokka.stubbing;
+
+import com.github.tomakehurst.wiremock.stubbing.StubMappings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.hycom.mokka.emulator.mock.MockConfigurationRepository;
+import pl.hycom.mokka.emulator.mock.model.MockConfiguration;
+
+import java.util.ArrayList;
+
+@ExtendWith(MockitoExtension.class)
+public class WireMockMappingSourceTest {
+
+    @InjectMocks
+    private WireMockMappingSource wireMockMappingSource;
+
+    @Mock
+    private MockConfigurationRepository mockConfigurationRepository;
+
+    @Mock
+    private StubMappings stubMapping;
+
+    @BeforeEach
+    private void setUp(){
+        Mockito.when(mockConfigurationRepository.findAll()).thenReturn(new ArrayList<MockConfiguration>());
+    }
+
+    @Test
+    public void whenMockConfigurationListIsEmpty(){
+        wireMockMappingSource.loadMappingsInto(stubMapping);
```

Comment:
Missing assertions. For example you could verify if there was no call to stubMapping.addMapping:
https://www.baeldung.com/mockito-verify